core-web/apps/dotcms-ui/src/app/view/components/dot-form-dialog/dot-form-dialog.component.spec.ts

```diff
@@ -93,6 +105,14 @@ describe('DotFormDialogComponent', () => {
             expect(component.save.emit).toHaveBeenCalledWith(event);
         });
 
+        // TODO: find a way to dispatch the event and that it gets caught by the listener
+        xit('should have emit save event on CMD + ENTER keys', () => {
+            const saveButton = de.query(By.css('#testInput'));
+            dispatchKeydownEvent(saveButton.nativeElement, 'enter', true);
```

Comment:
I think the problem might be related with that you are sending the event from a button but the listener in the component is tied to the document.

Also can you please check those listener? I think the subscribres are not being unsubscribe.